src/qml/qgismobileapp.qml

```diff
@@ -258,15 +283,15 @@ ApplicationWindow {
 
       property VectorLayer currentLayer: dashBoard.currentLayer
 
-      rubberbandModel: digitizingRubberband.model
+      rubberbandModel: stateMachine.state === 'measure' ? measuringRubberband.model : digitizingRubberband.model
       project: qgisProject
       crs: qgisProject.crs
     }
 
     x: mainWindow.width / 2 + 24 * dp
     y: mainWindow.height / 2 + 24 * dp
 
-    text: qfieldSettings.numericalDigitizingInformation && stateMachine.state === "digitize" ?
+    text: ( qfieldSettings.numericalDigitizingInformation && stateMachine.state === "digitize" ) || stateMachine.state === 'measure' ?
```

Comment:
do we really want to dispay the coords all the time?
we could maybe display it before we actually have a line (i.e. you are measure a point until you start measuring distance/area).

Reply:
Hm... or having the coords in case of the `numericDigitizingInformation` setting?

Reply:
one or the other yes, it might still be interesting to display them without having to change the setting. but no strong opinion

Reply:
Hm... I like your idea as well. I'll think about it...

Reply:
What's the downside of showing it?
Would it be better to move it to the end instead of the beginning?

Reply:
It's to make it slimmer. Do you think the coordinates are requested? 

Like I said I assume it's not really needed 

> the coordinates can be helpful on "finding the next vertex to digitize" but on measure, you know the point already

But I dont mind to put it back. Just tell me.

Reply:
It's not a request, but I still think it could be nice. Maybe make them in gray, to reduce their visual weight?